### PR TITLE
build(ci): use travis scripts for ci instead of ci.sh

### PR DESCRIPTION
**Pull request by priyajeet**
1. Removes ci.sh script
2. Caches node_modules on travis
3. Parallelizes ci build tasks
4. Adds babel build
5. Removes externalized react webpack build

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @priyajeet has signed the CLA. Thanks for the pull request!

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,21 @@
...
+cache:
+  yarn: true
+  directories:
+    - node_modules
+jobs:
+  include:
+    - # commit lint
+      script: commitlint-travis
+    - # code lint
+      script: npm-run-all clean build:i18n lint
```
Comment (DanDeMicco) [inline .travis.yml]: How long does `build:i18n` take? Is it possible to cache that directory?

**Comment by boxcla** [comment]
# Webpack Bundle Size Github Bot

| Bundle        | Diff | Percentage Diff | Before | After |
| ------------- |:----:|:---------------:|:------:|:-----:|
| Total | 0 B | 0.00% | 0 B | 0 B |

Compared against previously merged PR #644

**Powered by [webpack-bundle-size-github-bot](https://github.com/wenboyu2/webpack-bundle-size-github-bot/)**

File: .travis.yml [hunk 2]
```diff
@@ -1,5 +1,21 @@
...
+    - # flow types
+      script: npm-run-all clean build:i18n; yarn flow
```
Comment (jstoffan) [inline .travis.yml]: Just checking, is the `;` before `yarn flow` intentional? If so, what does it do and why here but not in the other `npm-run-all` scripts?

Comment (priyajeet) [reply .travis.yml]: ; is same as && for shell

File: .travis.yml (hunk 1 above)
Comment (priyajeet) [reply .travis.yml]: Pretty fast, less than 5s

**Pull request merged**